airbyte_cdk/sources/declarative/manifest_declarative_source.py

```diff
@@ -58,7 +58,12 @@
 
 
 class ManifestDeclarativeSource(DeclarativeSource):
-    """Declarative source defined by a manifest of low-code components that define source connector behavior"""
+    """Declarative source defined by a manifest of low-code components that define source connector behavior
+
+    If any stream in the source uses a DynamicSchemaLoader, config validation will be skipped during
+    discovery. This allows sources with dynamic schemas to run discovery without requiring authentication
+    when the schema endpoint doesn't need auth to provide catalog information.
+    """
```

Comment:
Reverting comment update. It is stated backwards and anyway isn't relevant to core class functionality.

```suggestion
    """Declarative source defined by a manifest of low-code components that define source connector behavior"""
```